src/style/global.sass

```
@@ -37,3 +37,12 @@ ul, ol
     font-size: 3rem
     margin-bottom: 1rem
     font-weight: bold
+
+code
+    font-family: $font-code
+    padding-inline: 0.5em
+    padding-block: 0.5em
+    font-size: 1.1em
+    direction: ltr //! doesn't work for some reason
+    color: $code-default-color
+    unicode-bidi: plaintext
```

Use `display: block` to cover all width
```suggestion
code
    font-family: $font-code
    padding: 1rem
    color: $code-default-color
    unicode-bidi: plaintext
    display: block
    background: $bg-color-secondary
    border-radius: 1rem
```